docs/ui/schema_viewer.rst

```diff
@@ -0,0 +1,69 @@
+.. _ref_ui_schema_viewer:
+
+=============
+Schema Viewer
+=============
+
+.. image:: images/schema_viewer.png
+    :alt: The Schema Viewer page in the EdgeDB UI. The icon is two small
+          squares of blue and orange, connected by a purple line. A small
+          user-defined sample schema is shown with two concrete types
+          called Book and Library, along with an abstract type called
+          HasAddress that is extended by the Library type.
+    :width: 100%
```

Comment:
I would move this image down to the "Graph view" section, just under that heading.